INFO     root:test_store.py:32 用户ID:4 模块:用户管理 场景:添加成功 标题:添加用户成功
INFO     root:analyse_data.py:28 1.解析请求数据，请求数据为{'method': 'post', 'url': 'http://192.168.10.131:8888/api/private/v1/users', 'headers': {'Authorization': 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1aWQiOjUwMCwicmlkIjowLCJpYXQiOjE3MzAzODA5OTAsImV4cCI6MTczMDQ2NzM5MH0.6e1B3whD82yyfV7LD_ZdekL62wtRKDHt6YlZOl_rYB8'}, 'params': None, 'json': None, 'data': {'username': 'potato', 'password': '123456'}, 'files': None}
INFO     root:send_request.py:15 2.发送http请求，响应数据为{'data': {'id': 562, 'username': 'potato', 'role_id': -1, 'create_time': 1730380990}, 'meta': {'msg': '创建成功', 'status': 201}}
INFO     root:asserts.py:14 3.http响应结果断言，预期结果 创建成功 ==实际结果创建成功